hack/test-unit.sh

```diff
@@ -10,6 +10,7 @@ PACKAGES_TO_TEST=(
     "github.com/openshift/console-operator/pkg/console/operator"
     "github.com/openshift/console-operator/pkg/console/starter"
     "github.com/openshift/console-operator/pkg/console/subresource/configmap"
+    "github.com/openshift/console-operator/pkg/console/subresource/consoleserver"
```

Comment:
This is a good one to bring back.
Tempting to separate it out into a separate PR, just in case backports of this PR come into play.

Reply:
Do you mean just to have the `config_builder_test.go` and `config_merger_test.go` disabled? cause we will need to update the `config_builder.go` with the necessary logic anyway